index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      @media screen and (min-width: 480px) {
+  #leftsidebar {width: 200px; float: left;}
+  #main {margin-left: 216px;}
+    </style>
+  </head>
+    
+  </head>
+
+  <body>
+
+    <nav id="navbar" >
+      <ul >
+        <ul>
+          <a href="#welcome-section">Welcome</a>
+        </ul>
+        <ul>
+          <a href="#project-tile">Projects</a>
+        </ul>
+        <ul>
+          <a href="#profiles">Connect</a>
+        </ul>
+      </ul>
+    </nav>
+
+
+    <section id="welcome-section">
+      <h1 >SAMUEL AND DIVAKAR</h1>
+      <p>Web Development Students</p>
+
+    </section>
+
+    <div><h1 id="works-font">HERE ARE OUR WORKS</h1></div>
+
+    <section id="projects">
+    <div class="tribute-tile" >
+      <a  id="t-tile" href="https://tribute-page.freecodecamp.rocks/" target="_blank"> 
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="">  
+
+        Tribute Page 
+      </a>
+    </div>
+    
+<div class="project-tile">
```

Comment:
Kindly pay attention 👀  to indentations as it needs to be consistent throughout your code. 🔆

Reply:
Good job 👍